data/Person.csv

```diff
@@ -43,62 +43,62 @@ P0021,Guo,Shiying,en,male,no,NULL,NULL,NULL,1942,1968,,,PL0012,S03,yes,T00011,75
 P0022,张,鹤慈,zh,male,no,NULL,NULL,NULL,1943,,,,,S03,yes,T00011,75f 80,精神重力與個人詞源 ——中國先鋒詩歌論 ,122,grandson of Zhang Dongsun 张东荪. Son of Zhang Zongbing 张宗炳. Was sent to a laogai for 15 years. He founded the X literary society and the X shouchao journal. ,pls add the source_2 in the text file,2017-07-05,LH,2018-01-18,OS
-P0023,Yang,Sanbai,en,,,,,,,,,,,,,T00011,75,,,,,2017-07-05,LH,2017-11-17,MG
-P0023,杨,三白,zh,,,,,,,,,,,,,T00011,75,,,,,2017-07-05,LH,2017-11-17,MG
-P0024,Song,Haiquan,en,male,yes,1969,,PL0022,1949,,,,,,,T00011,75 79f,http://club.kdnet.net/dispbbs.asp?id=2621201&boardid=2&page=6&uid=&usernames=&userids=&action=,ONLINE SOURCE,,,2017-07-05,LH,2018-01-16,OS
-P0024,宋,海泉,zh,male,yes,1969,,PL0022,1949,,,,,,,T00011,75 79f,http://club.kdnet.net/dispbbs.asp?id=2621201&boardid=2&page=6&uid=&usernames=&userids=&action=,ONLINE SOURCE,,,2017-07-05,LH,2018-01-16,OS
+P0023,Yang,Sanbai,en,,unknown,NULL,NULL,,,,,,,,,T00011,75,,,,,2017-07-05,LH,2018-01-25,WH
+P0023,杨,三白,zh,,unknown,NULL,NULL,,,,,,,,,T00011,75,,,,,2017-07-05,LH,2018-01-25,WH
+P0024,Song,Haiquan,en,male,yes,1969,unknown,PL0022,1949,,,,,,,T00011,75 79f,http://club.kdnet.net/dispbbs.asp?id=2621201&boardid=2&page=6&uid=&usernames=&userids=&action=,ONLINE SOURCE,,,2017-07-05,LH,2018-01-25,WH
+P0024,宋,海泉,zh,male,yes,1969,unknown,PL0022,1949,,,,,,,T00011,75 79f,http://club.kdnet.net/dispbbs.asp?id=2621201&boardid=2&page=6&uid=&usernames=&userids=&action=,ONLINE SOURCE,,,2017-07-05,LH,2018-01-25,WH
 P0025,Gan,Tiesheng,en,male,yes,1968,1974,PL0141,1946,,Zifeng,en,PL0002,S03,,T00011,75-76  80,http://baogaowenxue.xiusha.com/z/zheyongping/stmq/004.htm,ONLINE SOURCE,,,2017-07-05,LH,2018-01-18,OS
 P0025,甘,铁生,zh,male,yes,1968,1974,PL0141,1946,,紫峰 ,zh,PL0002,S03,,T00011,75-76  80,http://baogaowenxue.xiusha.com/z/zheyongping/stmq/004.htm,ONLINE SOURCE,,,2017-07-05,LH,2018-01-18,OS
 P0026,Lu,Shuangqin,en,female,yes,1969,,PL0008,1953,,,,PL0002,,Yes,T00011,76f,https://www.jintian.net/today/html/28/n-17928.html,ONLINE SOURCE,,,2017-07-05,LH,2018-01-10,OS
 P0026,鲁,双芹,zh,female,yes,1969,,PL0008,1953,,,,PL0002,,Yes,T00011,76f,https://www.jintian.net/today/html/28/n-17928.html,ONLINE SOURCE,,,2017-07-05,LH,2018-01-10,OS
-P0027,Zhang,Xinhua,en,,,,,,,,,,,,,T00011,76 80,,,,,2017-07-05,LH,2017-11-17,MG
-P0027,张,新华,zh,,,,,,,,,,,,,T00011,76 80,,,,,2017-07-05,LH,2017-11-17,MG
-P0028,Zhang,Jiuxing,en,,,,,,,"[1972, 1973]",,,,,,T00011,76,,,,,2017-07-05,LH,2018-01-09,OS
-P0028,张,久兴,zh,,,,,,,"[1972, 1973]",,,,,,T00011,76,,,,,2017-07-05,LH,2018-01-09,OS
+P0027,Zhang,Xinhua,en,male,unknown,NULL,NULL,,,,,,,,,T00011,76 80,,,,,2017-07-05,LH,2018-01-25,WH
+P0027,张,新华,zh,male,unknown,NULL,NULL,,,,,,,,,T00011,76 80,,,,,2017-07-05,LH,2018-01-25,WH
+P0028,Zhang,Jiuxing,en,male,unknown,NULL,NULL,,,"[1972, 1973]",,,,,,T00011,76,,,,,2017-07-05,LH,2018-01-25,WH
```

Comment:
Out of curiosity: How did you learn about the sex of Zhang Xinhua and Zhang Jiuxing?

Reply:
Perfect, thank you! 👍